inc/post-types.php

```diff
@@ -2444,6 +2449,24 @@ function cptui_updated_post_type_slug_exists( $slug_exists, $post_type_slug = ''
 }
 add_filter( 'cptui_post_type_slug_exists', 'cptui_updated_post_type_slug_exists', 11, 3 );
 
+/**
+ * Ignores the slug validation for an existing CPT if the override checkbox was previously selected.
+ *
+ * @since 1.14.0
```

Comment:
can you change this to "NEXT" or similar? I pushed out 1.14.0 earlier this morning.